code-for-canada/project-thundercat
# [Team Information] Populating team info tree view popup with db data

# Description

This PR is replacing localized variables in organizational structure image description popup with DB data.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**English Version:**

![image](https://user-images.githubusercontent.com/23021242/60671534-4b021000-9e41-11e9-9c37-7b4e3a5e4f05.png)

**French Version:**

![image](https://user-images.githubusercontent.com/23021242/60671556-548b7800-9e41-11e9-8744-7789354741ce.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2. Click on Background tab.
3.  Click on Your team_ on side nav.
4.  Open the image description.
5.  Make sure that the popup content looks exactly the same as before in both languages.

# Checklist
